backend/pkg/api/enums/validator_dashboard_enums.go

```diff
@@ -170,6 +170,35 @@ var VDBBlocksColumns = struct {
 	VDBBlockProposerReward,
 }
 
+type VDBBlocksSearches struct {
```

Comment:
Don't know if these structs fit in the enum package. But also don't know where else they'd fit.

Reply:
I moved it to `types`, I think that's slightly better, ptal